test/util/IdTableHelpers.h

```diff
@@ -48,15 +49,12 @@ IdTable makeIdTableFromVector(const VectorTable& tableContent) {
   for (const auto& row : tableContent) {
     AD_CHECK(row.size() == result.numColumns());  // All rows of an IdTable must
                                                   // have the same length.
-    const size_t backIndex{result.size()};
-    // TODO This should be
-    // std::ranges::copy(std::views::transform(row, I), result.back().begin());
-    // as soon as our compilers and the IdTable support it.
+    // Create a new empty row at the bottom of the IdTable.
     result.emplace_back();
-
-    for (size_t c = 0; c < row.size(); c++) {
-      result(backIndex, c) = I(row[c]);
-    }
+    // transform() creates a view, which consits of the returned values of I
+    // , after calling it with each element of row. This view is then copied
+    // into the empty row at the bottom of IdTable.
+    std::ranges::copy(std::ranges::views::transform(row, I), result.back().begin());
```

Comment:
This unfortunately doesn't work yet, because clang has problems with all the `std::views` stuff. 
So we have to leave this as is and update the `TODO` to start with `TODO<clang16>` and change the text to "as our compilers support it".

Reply:
Done, see [656b045](https://github.com/ad-freiburg/qlever/pull/865/commits/656b04510783a0dc95d39adf4dce34af4cc3646c).